t/03-testapi.t

```diff
@@ -367,6 +367,13 @@ subtest 'script_run' => sub {
     $fake_exit = 1234;
     is(background_script_run('sleep 10'), '1234', 'background_script_run returns a PID');
     is(background_script_run('sleep 10', output => 'foo'), '1234', 'background_script_run with output returns valid PID');
+
+    my $mock_testapi = Test::MockModule->new('testapi');
+    $mock_testapi->redefine(is_serial_terminal => 1);
+    $mock_testapi->redefine(wait_serial => 'XXXfoo-SCRIPT_FINISHEDXXX');
+
+    throws_ok { background_script_run('foo') } qr /PID marker not found/, 'dies without PID marker';
```

Comment:
Using `qr /.../` with a space is a bit uncommon. It's possible, but not really common in our code, so I would suggested to remove the space.